apps/sertra/components/Layout.module.css

```diff
@@ -0,0 +1,6 @@
+.main {
+  width: 100%;
+  max-width: var(--max-width-content);
```

Comment:
This does the job, but I feel it would be good to keep it consistent across the board; now `<header>` and `<main>` both have the layout width defined on themselves, but footer has a separate inner container (because of the background-color requirement). I myself usually tend to lean towards the latter for flexibility.

That being said, in any case the layout structure right now is very different from the upcoming new UI despite having basically the same elements and structure. That's probably because we are not using the new components here right now, but still I feel it would make sense to build them the same way. So, not a blocker but something to consider moving forward.

Reply:
Didn't do anything concrete based on this comment.